modules/vf-graphql-holochain/queries/process.ts

```diff
@@ -9,15 +9,20 @@ import { DNAIdMappings, ReadParams } from '../types.js'
 import { mapZomeFn } from '../connection.js'
 
 import {
-  Process, ProcessResponse,
+  Process, ProcessConnection, ProcessResponse,
 } from '@valueflows/vf-graphql'
+import { PagingParams } from '../resolvers/zomeSearchInputTypes.js'
 
 export default (dnaConfig: DNAIdMappings, conductorUri: string) => {
   const readOne = mapZomeFn<ReadParams, ProcessResponse>(dnaConfig, conductorUri, 'observation', 'process', 'get_process')
+  const readAll = mapZomeFn<PagingParams, ProcessConnection>(dnaConfig, conductorUri, 'observation', 'process_index', 'read_all_processs')
```

Comment:
did we want to change the macro a bit so that the function is `read_all_processes` instead of `read_all_processs`?

Reply:
@pospi

Reply:
Sure. I already have a proc macro attribute for the query method name, the read all method name value is just an oversight.

You can see in the process index zome it's declared as `#[index_zome(query_fn_name="query_processes")]` so should have a pattern to follow to add another.

Reply:
done in 639a5e82